scripts/buildall.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+./sbt -J-Xmx2G "project vectortile" test || { exit 1; }
```

Comment:
need to also add to a `.travis/build-and-test-set` file, or else it won't be tested by travis